content/docs/bdd/discovery-workshop.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Discovery workshop
+subtitle: The session where you do your exploration work
+---
+
+### What is a discovery workshop?
+
+A discovery workshop is a conversation where technical and business people collaborate to explore, discover and agree as much as they can about the desired behaviour for a [User Story](/docs/terms/user-story/).
+
+### How do you conduct a discovery workshop?
+
+There are several discovery workshop models, these are just a few:
+
+[**Example Mapping**](/docs/bdd/example-mapping/)
+
+Uses a pack of index cards in four different colours to map rules (a summary of constraints/acceptance criteria the team has agreed upon) to examples (illustrations/cases of the acceptance criteria)
+
+[**OOPSI Mapping (Outcomes, Outputs, Processes, Scenarios, Inputs)**](https://jennyjmar.com/2016/04/16/bdd-discovery-and-oopsi/)
+
+Similar to Example Mapping, uses Post-it Notes of different colours to map shared processes/relationships between outputs and scenarios.
+
+[**Feature Mapping**](https://johnfergusonsmart.com/feature-mapping-a-simpler-path-from-stories-to-executable-acceptance-criteria/)
+
+Also uses Post-it Notes of different colours.  The team picks a story from the backlog, identifies the actors involved, breaks the story down into tasks, and maps those tasks to specific examples.
+
+### When should you hold a discovery workshop?
+
+As late as possible before development on a new User Story begins, in order to prevent details from being lost. Conducting a discovery workshop as late as possible also gives the team enough leg room to shift their plans should new details surface.
+
+### Who should attend?
+
+A good rule-of-thumb is 3-6 people, but at a bare minimum your [Three Amigos](/docs/bdd/who-does-what/#the-three-amigos/) should be present:  a product owner, a developer, and a tester.  Your product owner will identify the problem the team should be trying to solve, your developer will address how to build a solution around said problem, and your tester will address any edge cases that could arise. 
```

Comment:
```suggestion
A good rule-of-thumb is 3-6 people, but at a bare minimum your [Three Amigos](/docs/bdd/who-does-what/#the-three-amigos/) should be present: a product owner, a developer, and a tester.  Your product owner will identify the problem the team should be trying to solve, your developer will address how to build a solution around said problem, and your tester will address any edge cases that could arise. 
```

Reply:
Double space here. (Sorry, I seem t have an eye for typos). You can fix it by accepting my suggestion.

Pro tip: Next time you do a PR, you could create a branch on the cucumber repo, rather than your own which would make it easier for us to just fix them directly ;)

Reply:
Hmm, I just noticed you use double spaces after a period too, so maybe this is by design. Not sure what the convention is through the docs, but let's pick one and stick with it (and add it to the guidelines for contributing to the docs)